### PR TITLE
Added DracthyrAlliance & DracthyrHorde Dragonflight character races.

### PR DESCRIPTION
**Pull request by WingsOfAltair**

### CODE REVIEW

**Comment by WingsOfAltair** [comment]
I will need some help implementing the DBC for TrinityCore master branch 10.1.0.50000 in the near future.

File: WoWDatabaseEditor.Common/WDE.Common/Database/CharacterRaces.cs [hunk 1]
```diff
@@ -44,5 +46,6 @@ public enum CharacterRaces : uint
     AllWoD = AllMoP,
     AllLegion = AllWoD | Nightborne | HighmountainTauren | VoidElf | LightforgedDraenei,
     AllBfA = AllLegion | ZandalariTroll | KulTiran | DarkIronDwarf | Vulpera | MagharOrc | Mechagnome,
-    AllShadowlands = AllBfA
```
Comment (BAndysc) [inline WoWDatabaseEditor.Common/WDE.Common/Database/CharacterRaces.cs:47]: Could you also add new faces to AllAlliance/AllHorde enum?

**Comment by BAndysc** [comment]
Good job, one small comment added, feel free to text me on discord for help regarding dbc reading!

**Comment by WingsOfAltair** [comment]
> Good job, one small comment added, feel free to text me on discord for help regarding dbc reading!

I couldn't find your Discord ID anywhere. Here's mine. SheikhHash#7594

File: WoWDatabaseEditor.Common/WDE.Common/Database/CharacterRaces.cs (hunk 1 above)
Comment (WingsOfAltair) [reply WoWDatabaseEditor.Common/WDE.Common/Database/CharacterRaces.cs:47]: > Could you also add new faces to AllAlliance/AllHorde enum?

Talk to me on Discord as I have a question regarding this matter. I have given you my Discord ID in a comment below.

**Pull request merged**